content/reference/custom_interactions.md

```diff
@@ -0,0 +1,221 @@
++++
+title = 'Custom Interactions'
+weight = 6
++++
+
+Buttons, Modals, Select Menus, Ephemeral Messages, and more!
+
+<!--more-->
+
+{{% notice warning %}}
+
+Use of interactions within YAGPDB is an advanced topic. You will need a thorough understanding of YAGPDB's scripting
+language before learning interactions.
+
+We have a comprehensive learning course located [here](https://learn.yagpdb.xyz) for you to get started.
```

Comment:
Wanted to note this down as a general style suggestion: we should avoid using "click here" or "located here" in links; see, e.g., <https://ux.stackexchange.com/questions/12100/why-shouldnt-we-use-words-such-as-here-and-this-in-textlinks> for reasoning. In this case I would prefer to write
```suggestion
We have a [comprehensive learning course](https://learn.yagpdb.xyz) for you to get started.
```

Reply:
Thanks, in my initial review I somehow missed that. I think we have a lot of similar things in the migrated documentation files as well; I will review those at a later date.